src/main/java/com/obal/dominos/App.java

```diff
@@ -1,19 +1,102 @@
 package com.obal.dominos;
 
-import java.util.Scanner;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedList;
 
-/**
- * Hello world!
- *
- */
+class Domino {
+    byte[] values;
+
+    public Domino(byte[] vl) {
+        values = vl;
+    }
+
```

Comment:
You usually override toString in order to be able to call `System.out.println(<domino instance>).` 
Similarly you could override `Hand.toString` using the created `Domino.toString`